packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.19
```

Comment:
I dont think you want this in your pr.

Reply:
Are you looking at the commit where I merged in `main` rather than the full PR?